urlchecker/client/check.py

```diff
@@ -89,14 +94,14 @@ def main(args, extra):
         delete_repo(path)
 
     # Case 1: We didn't find any urls to check
-    if not check_results['failed'] and not check_results['passed']:
+    if not check_results["failed"] and not check_results["passed"]:
         print("\n\nDone. No urls were collected.")
         sys.exit(0)
 
     # Case 2: We had errors, but force pass is True
-    elif args.force_pass and check_results['failed']:
+    elif not args.force_pass and check_results["failed"]:
         print("\n\nDone. The following urls did not pass:")
-        for failed_url in check_results['failed']:
+        for failed_url in check_results["failed"]:
             print_failure(failed_url)
         sys.exit(1)
 
```

Comment:
This will work fine but the final output is very helpful and the force pass is more about the `sys.exit()`. Wouldn't it make sense, to keep the print and just change the exit status?

Reply:
What do you mean? The print is still there, the change just shows changing the quotation style.

Reply:
no I mean in case there is force pass is true, and there are failed links, the urlchecker will force the pass but no urls are printed, which actually helpful to see imo. idk if is that big, might be a matter of preference. My idea is that the force pass is just to exit with  1 and it should not change the prints.

Reply:
Oh I understand! Yes let me fix this now. You'll need to approve it again so don't approve it again yet :)

Reply:
okay perfect ;)

Reply:
btw are you using a linter?

Reply:
okay take a look!

Reply:
Yes, I think I might have run it accidentally, but the code is so neat the changes were very minimal!

Reply:
I am just asking because I noticed that we use different styles and it might be good for the future to figure out a general standard ;)

Reply:
I used to have my own weird style many years ago, and then I was using pylint for a while (and it was quite arduous) and when I noticed snakemake using black, and then tried it myself and thought that it made the code lovely, I started using black. It does have minor changes between versions, but it's so easy to use and add to CI I use it for most of my python projects. Locally I just do:

```
black <project>
```
and then in CI you can do:

```
black --check <project>
```

Reply:
I personally like https://github.com/google/yapf but I also have my hybrid style based on some weird preferences that I am trying to get rid of xD I will take a look at black -which is trending these days-  and see how is it.